App.js

```diff
@@ -14,6 +16,24 @@ import NewsScreen from './Screens/NewsScreen';
 const Stack = createNativeStackNavigator();
 
 function App() {
+
+  const [loaded, error] = useFonts({
+    Montserrat_500Medium,
+    Montserrat_700Bold,
+  });
+
+  if (!loaded && !error) {
+    return null;
+  }
+
+  if (error) {
+    return (
+      <View style={styles.errorContainer}>
+        <Text style={styles.errorText}>Error loading fonts</Text>
+      </View>
+    );
+  }
```

Comment:
NIT: I think it's fine to have these specific error pages for debugging, however this might be pretty weird to show to a user in the final version. Maybe in the future we can have a general error page `<ErrorPage />` to show users to display on import errors.

Reply:
Agreed